grin/test/AbstractInterpretation/ExtendedSyntax/LiveVariableSpec.hs

```diff
@@ -31,6 +31,9 @@ import AbstractInterpretation.ExtendedSyntax.EffectTrackingSpec (calcEffects)
 
    Variables with names like "_<i>" are introduced just for named bindings.
    They will never be used.
+
+   Variables with names like "alt<i>" are introduced just for named case alternatives.
+   They will never be used.
```

Comment:
They will never be used (yet)

Reply:
This is just a naming convention in the tests.